src/components/Card/HpDmgFooter/hpDmgFooter.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: 'robotoRegular';
+  src: url('/src/assets/fonts/Roboto-Regular.ttf') format('truetype'); 
```

Comment:
this is an the same as in description.css, iirc there is a way to atleast import/define src of the font without repeating it in 2 places